practitioner-service/src/main/resources/bootstrap-dev.properties

```diff
@@ -1,6 +1,6 @@
 spring.application.name=practitioner-service
 
-spring.cloud.config.uri=http://localhost:8888
+spring.cloud.config.uri=http://placeholder:port
 
 spring.cloud.config.profile=default
```

Comment:
So the app is running in dev profile but pulling config from the config-server under the default profile. Seems a bit confusing...

Reply:
typo

Reply:
it worked both in my tests and when deployed since env vars take precedence, so in fact it wasn't used (the value mentioned here)